src/styles/_card.scss

```
@@ -0,0 +1,80 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+  transition-duration: $transitionduration;
+  transition-timing-function: linear;
```

You want to specify which properties are transition. Btw you can use a shorthand
```suggestion
  transition: transfrom $transitionduration linear;
```